README.md

```diff
@@ -29,6 +29,23 @@ make
 sudo make install
 ```
 
+### Testing:
+
+You can run test for runC by using command:
+
+```bash
+# make test
+```
+
+Note that test cases are run in Docker container, so you need to install
+`docker` first.
```

Comment:
The last I heard (although I haven't been following this closely) there were [minimum version requirements for Docker compatibility][1].  We might want to list any such requirements here.

[1]: https://github.com/opencontainers/runc/issues/186#issuecomment-128523088